webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java

```diff
@@ -217,8 +217,7 @@ public ScrambleRequest(String title, String scrambleRequestUrl, String seed) thr
         return scrambleRequests;
     }
 
-
-    private static PdfReader createPdf(String globalTitle, Date creationDate, ScrambleRequest scrambleRequest) throws DocumentException, IOException {
+    private static PdfReader createPdf(String pdfFileName, String globalTitle, Date creationDate, ScrambleRequest scrambleRequest) throws DocumentException, IOException {
```

Comment:
Why did you add `pdfFileName` as an argument to this function? It looks like it's not used anywhere in this method.